app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/roadbook/RoadBookFragmentTest.kt

```diff
@@ -71,7 +72,7 @@ class RoadBookFragmentTest {
     fun toRoadBookFragment() {
         // Ensure "use RoadBook preferences" is disabled
         PreferencesSetting.setRoadBookPrefs(testRule)
-
+        GeneralUtils.fillUserEntryAndEnterTheApp("boss@gmail.com", "123456")
         onView(withId(R.id.drawer_layout))
```

Comment:
For the RoadBook, the following three lines could be removed, as it is the first Fragment after the login entry.

Also remark that for all other Fragment, a wrong user-entry will still being bypassed by an open action on the DrawerMenu, but I don't think it is a problem for our before routine.